libcore/FileUtils.vala

```diff
@@ -552,6 +552,10 @@ namespace PF.FileUtils {
                 return false;
         }
     }
+
+    public bool is_icon_path (string path) {
```

Comment:
This should be static.

Reply:
As per Slack discussion, methods outside classes are implicitly static.  If required for elementary code style, then I will submit a separate PR explicitly designating all the FileUtils methods to be static.